controllers/fyController.js

```diff
@@ -0,0 +1,314 @@
+      if (err) reject(err);
+      else resolve(res);
+    })
+  })
+};
+
+//reset the Fy table
+exports.resetFy = async (req, res) => {
+  const query = 'DELETE FROM students;';
+
+  const reset = new Promise((resolve, reject) => {
+    connection.query(query, (err, res) => {
+      if (err) reject(err);
+      else resolve(res);
+    })
+  });
+
+  try {
+    await reset;
+  } catch (error){
+    return res.send({ status: STATUS_CODE.ERROR, result: error });
+  }
+
+  return res.send({ status: STATUS_CODE.SUCCESS });
+};
+
+//load with csv file
+exports.fyLoadfromcsv = async (req, res) => {
+  const { file } = req.body;
+  var duplicates = [];
+
+  // Fetching the data from each row 
+  // and inserting to the table "sample"
+  for (var i = 0; i < file.length; i++) {
+    var email = file[i]["email"],
+      name = file[i]["name"],
+      visions = file[i]["visions"];
+
+    try {
+      let verify = await this.verifyFy( email );
+      if (verify.NUM > 0) {
+        duplicates.push(email);
+      } else{
+        await this.insertFy({ email, name, visions });
+      }
+    } catch (error) {
+      return res.send({ status: STATUS_CODE.ERROR, result: error });
+    }
+  }
+
+  if(duplicates.length == 0){
+    return res.send({ status: STATUS_CODE.SUCCESS });
+  } else{
+    return res.send({ status: STATUS_CODE.EMAIL_USED, result: duplicates });
+  }
+};
+
+//add one Fy
+exports.createFy = async (req, res) => {
+  const { email, name, visions } = req.body;
+
+  try {
+    let verify = await this.verifyFy( email );
+    if (verify.NUM > 0) {
+      return res.send({ status: STATUS_CODE.EMAIL_USED, result: email });
+    }
+
+    let result = await this.insertFy({ email, name, visions });
+    
+    if (result.affectedRows) {
+      return res.send({ status: STATUS_CODE.SUCCESS });
+    }
+  } catch (error) {
+    return res.send({ status: STATUS_CODE.ERROR, result: error });
+  }
+
+  return res.send({ status: STATUS_CODE.ERROR });
+};
+
+//edit one Fy
+exports.updateFy = async (req, res) => {
+  const { old_email, email, name, visions } = req.body;
+
+  try{
+    let result = await this.editFy({ old_email, email, name, visions });
+    return res.send({ status: STATUS_CODE.SUCCESS });
+  } catch (error){
+    return res.send({ status: STATUS_CODE.ERROR, result: error });
+  }
+
+};
+
+//delete one Fy
+exports.deleteFy = async (req, res) => {
+  try{
+    const email = req.body.email;
+   
+    let verify = await this.verifyFy( email );
+
+    if (verify.NUM == 0) {
+      return res.send({ status: STATUS_CODE.INCORRECT_STUDENT_EMAIL, result: email });
+    }
+
+    let result = await this.removeFy( email );
+
+    if (result.affectedRows){
+      return res.send({ status: STATUS_CODE.SUCCESS });
+    }
+  } catch (error) {
+    return res.send({ status: STATUS_CODE.ERROR, result: error });
+  }
+  return res.send({ status: STATUS_CODE.SUCCESS});
+};
+
+//get all first year students, return a json object
+exports.readFy = async (req, res) => {
```

Comment:
When I change row_start, 'pages' is returned with a value 0.
<img width="783" alt="image" src="https://user-images.githubusercontent.com/107736505/197261935-1ed3a975-ed1f-4cd0-bba8-bfdedb6b3bfd.png">